### PR TITLE
[CDAP-16451][CDAP-16457] Fix Configurable tab js error in UI

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA:
https://issues.cask.co/browse/CDAP-16451
https://issues.cask.co/browse/CDAP-16457

Build: 
https://builds.cask.co/browse/CDAP-UDUT549-1

### CODE REVIEW

File: cdap-ui/server/url-helper.js [hunk 1]
```diff
@@ -38,16 +38,21 @@ function extractMarketUrls(cdapConfig) {
     return [defaultMarketUrl];
   }
   const marketUrls = cdapConfig['market.base.urls'].split('+');
-  const filteredMarketUrls = marketUrls.filter(element => element !== defaultMarketUrl); 
+  let filteredMarketUrls;
+  if (!defaultMarketUrl) {
```
Comment (yukiej) [inline cdap-ui/server/url-helper.js:46]: If it is possible for there to be no defaultMarketUrl, could this sometimes cause an issue on line 38?

Comment (ajainarayanan) [reply cdap-ui/server/url-helper.js:46]: The assumption is if the market.base.urls is not present then the market.base.url is assumed to be present. But you are right there can be corner case when both of them are not present which will render the hub useless. Anyhow I have added additional check.

**Review by yukiej — APPROVED** [review]
lgtm!

**Comment by ajainarayanan** [comment]
Green build: https://builds.cask.co/browse/CDAP-UDUT549-4

**Pull request merged**